cdap-ui/.eslintrc.json

```diff
@@ -1,25 +1,39 @@
 {
+  "parser": "babel-eslint",
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 7,
     "sourceType": "module",
     "ecmaFeatures": {
       "jsx": true
     }
   },
   "rules": {
+    "strict": 0,
+    "no-unused-vars": [2, {"vars": "local"}],
     "semi": 2,
-    "no-console": 0
+    "no-console": 0,
+    "babel/generator-star-spacing": 1,
+    "babel/new-cap": 0,
+    "babel/array-bracket-spacing": 1,
+    "babel/object-curly-spacing": 0,
+    "babel/object-shorthand": 0,
+    "babel/arrow-parens": 0,
+    "babel/no-await-in-loop": 1,
+    "babel/flow-object-type": 1,
+    "babel/func-params-comma-dangle": 1
   },
   "env": {
     "browser": true,
     "commonjs": true,
-    "node": true
+    "node": true,
+    "es6": true
```

Comment:
Talked to @elfenheart . There is no es7 environment currently available in eslint.